examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,44 @@
+"""
+Using distributed for single_machine parallel computing
+========================================================
+
+Realistic usage scenario: combining dask code with joblib code, for
+instance using dask for preprocessing data, and scikit-learn for machine
+learning.
```

Comment:
Would you consider "prototyping a solution, to later be run on a truly distributed cluster" a "realistic usage scenario".

That (prototyping, before moving to a cluster) and the diagnostics dashboard are my two most common use cases for using the distributed scheduler on a single-machine.